CHANGELOG.md

```diff
@@ -91,6 +91,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fix weighted routing metadata deserialization error on process restart ([#4691](https://github.com/opensearch-project/OpenSearch/pull/4691))
 - Refactor Base Action class javadocs to OpenSearch.API ([#4732](https://github.com/opensearch-project/OpenSearch/pull/4732))
 - Migrate client transports to Apache HttpClient / Core 5.x ([#4459](https://github.com/opensearch-project/OpenSearch/pull/4459))
+- Changed http code on create index API with bad input raising NotXContentException to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
```

Comment:
What was the old code (say changed from X to 400)

Reply:
Done